contacts.js

```diff
@@ -107,4 +96,46 @@ function addContact(name, email, phone) {
   });
 }
 
-module.exports = { invokeAction };
+function updateContact(contactId, name, email, phone) {
+  fs.readFile(contactsPath, { encoding: "utf-8" }, (err, data) => {
+    if (err) {
+      return console.error(err);
+    }
+
+    const space = "  ";
+    if (data.length < 1) data = "[]";
+    let parser = JSON.parse(data);
+
+    let par = parser.map(pa => {
+      let temp = pa.id;
+      if (typeof temp === "number") temp = JSON.stringify(temp);
+      if (temp === contactId) {
+        pa = `\n${space}{ \n  ${space}"id": "${pa.id}", \n  ${space}"name": "${name}", \n  ${space}"email": "${email}", \n  ${space}"phone": "${phone}" \n${space}}`;
+        return JSON.parse(pa);
+      }
+      return pa;
+    });
+
+    par = par.map(pa => {
+      pa = `\n${space}{ \n  ${space}"id": "${pa.id}", \n  ${space}"name": "${pa.name}", \n  ${space}"email": "${pa.email}", \n  ${space}"phone": "${pa.phone}" \n${space}}`;
```

Comment:
remove all code like this `\n${space}{ \n  ${space}"id": "${pa.id}", \n  ${space}"name": "${pa.name}", \n  ${space}"email": "${pa.email}", \n  ${space}"phone": "${pa.phone}" \n${space}}`;